alpha/declcfg/declcfg.go

```diff
@@ -100,25 +98,27 @@ func (m Meta) MarshalJSON() ([]byte, error) {
 }
 
 func (m *Meta) UnmarshalJSON(blob []byte) error {
-	blob = bytereplacer.New(`\u003c`, "<", `\u003e`, ">", `\u0026`, "&").Replace(blob)
-
-	type tmp struct {
-		Schema     string              `json:"schema"`
-		Package    string              `json:"package,omitempty"`
-		Name       string              `json:"name,omitempty"`
-		Properties []property.Property `json:"properties,omitempty"`
+	blobMap := map[string]interface{}{}
+	if err := json.Unmarshal(blob, &blobMap); err != nil {
+		return err
+	}
+	for key, ptr := range map[string]*string{"schema": &m.Schema, "package": &m.Package, "name": &m.Name} {
```

Comment:
Do we care about about processing the `properties` field as well? What happens with this implementation if the meta object's YAML has a populated `properties` field?

Reply:
In the original code, we are parsing to check for expected syntax of the properties field of _all_ schema types, regardless of whether its a known type or not. Then we throw those parsed properties away.

I'm not sure we should require _all_ schemas that want to have a properties field to follow our properties schema. Ideally, we could have this discussion separately from this PR. We could do that, but it would require a bit more complexity and resource utilization (potentially requiring multiple parsing passes) in this function.

Reply:
Fair enough, I'm definitely open to having that discussion separately from this PR